modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsFieldsDisplayContext.java

```diff
@@ -122,6 +122,18 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems()
 			fdsActionDropdownItem.setTarget("async");
 		}
 
+		ObjectDefinition objectDefinition = getObjectDefinition();
+
+		if (objectDefinition.isApproved() && objectDefinition.isModifiable() &&
+			objectDefinition.isSystem()) {
+
+			return Arrays.asList(
+				new FDSActionDropdownItem(
+					getEditObjectFieldURL(), "view", "view",
+					LanguageUtil.get(objectRequestHelper.getRequest(), "view"),
+					"get", null, "sidePanel"));
+		}
```

Comment:
@mateussgn you don't need to this,

you just need to make sure the delete action will be retrieved as null [here](https://github.com/liferay/liferay-portal/blob/357f569f2ac814e50cf64a624b830a840748e576/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L332-L343),

Should we adapt the method [objectField.isDeletionAllowed](https://github.com/liferay/liferay-portal/blob/357f569f2ac814e50cf64a624b830a840748e576/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectFieldResourceImpl.java#L334) as well?

Reply:
Great catch! Thank you!